AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -753,7 +753,7 @@ fonts.google.com#%#//scriptlet('trusted-set-local-storage-item', 'bannersDismiss
 /terms-cookies-content$domain=antallaktikaexartimata.gr|autoalkatreszek24.hu|autoczescionline24.pl|autodeleshop.dk|autodielyonline24.sk|autoonderdelen24.be|autoteiledirekt.de|autovaruosadonline.ee|besteonderdelen.nl|onlinecarparts.co.uk|pecasauto24.pt|piecesauto.fr|piecesauto24.lu|recambioscoche.es|reservdelar24.se|reservedeler24.co.no|rezervesdalas24.lv|teile-direkt.at|topautoosat.fi|tuttoautoricambi.it
 virginaustralia.com#%#//scriptlet('trusted-click-element', '#cookieAcceptanceBtn[class*="__button--"]')
 orcid.org##app-banners > app-banner:has(> .alert-banner > p > span a[href="https://orcid.org/privacy-policy"])
-abetterrouteplanner.com##.r-13awgt0 > div[style].r-18u37iz.r-1awozwy
+abetterrouteplanner.com##.r-13awgt0 > div[style].r-18u37iz.r-1awozwy.r-1dzdj1l
```

Comment:
https://github.com/AdguardTeam/AdguardFilters/commit/bfce51e01c3611eb84f3293eda378d1472619927
@piquark6046 Please be careful with such rules.

![image](https://github.com/user-attachments/assets/761bd7e0-02c6-4099-a2e0-79b21fc421d4)